public/index.html

```diff
@@ -2,7 +2,7 @@
 <html lang="en">
   <head>
     <meta charset="utf-8" />
-    <link rel="icon" href="%PUBLIC_URL%/psa-favicon.ico" />
+    <link rel="icon" href="%PUBLIC_URL%/ncsu-favicon.ico" />
     <meta name="viewport" content="width=device-width, initial-scale=1" />
     <meta name="theme-color" content="#000000" />
     <meta
```

Comment:
description + title needs to be updated
Will have to discuss tomorrow in drone meeting